src/main/java/io/meeds/qa/ui/pages/page/factory/space/SpaceHomePage.java

```diff
@@ -1215,4 +1214,40 @@ public void viewAllRepliesInCommentsDrawer(String comment) {
     getCommentsDrawerViewAllReplies(comment).clickOnElement();
   }
 
+  private BaseElementFacade getReactionActivityList(String activity) {
+    return findByXPathOrCSS(String
+            .format("(//*[contains(text(),'%s')]//following::p[contains(@class,'likersNumber')])[1]", activity));
+  }
+
+  public void clickReactionList(String activity) {
+    getReactionActivityList(activity).clickOnElement();
+  }
+
+  @FindBy(xpath = "//*[contains(@class,'v-list-item__content drawerTitle') and contains (text(),'Reactions')]")
```

Comment:
Please avoid using a Vuetify CSS helper or internal classes only when it's a comportemental class like '.v-window--active' or '.v-navigation-drawer--open'.
In addition, like explained in a comment below, this element is not useful in fact.

Reply:
useless element is removed